src/platforms/flutter/upload-debug.mdx

```diff
@@ -18,32 +18,134 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
+
+`url`
+
+: The URL of your project e.g. `https<area>://mysentry.invalid/`. This is a `string` type. The alternative environmental variable is `SENTRY_URL`.
+
+`wait_for_processing`
+
+: Wait for server-side processing of uploaded files. This is a boolean type with default value `false`.
+
+`log_level`
+
+: Configures the log level for sentry-cli. This is a `string` type with default value `warn`. The alternative environmental variable is `SENTRY_LOG_LEVEL`. Possible values are `trace`, `debug`, `info`, `warn`, and `error`.
+
+`release`
+
+: The release version for source maps, it should match the release set by the SDK. This is a `string` type with default value `name@version` from pubspec. The alternative environmental variable is `SENTRY_RELEASE`.
+
+`web_build_path`
+
+: The web build folder. This is a `string` type with default value `build/web`.
+
+`commits`
+
+: Release commits integration. This is a `string` type with default value `auto`.
+
+`ignore_missing`
+
+: Ignore missing commits previously used in the release. This is a boolean type with default value `false`.
+
+### Run
+
+```bash
+flutter packages pub run sentry_dart_plugin
+```
+
+### Troubleshooting
+
+Refer to [Troubleshooting - Sentry Dart Plugin](/platforms/flutter/troubleshooting#sentry-dart-plugin) to resolve potential issues.
+
+## Manually Upload Debug Symbols
+
+### Uploading for iOS and macOS
 
 Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. [Learn how to upload the dSYM files](/platforms/apple/dsym/).
 
 If you use the `split-debug-info` and `obfuscate` flags, you need to upload the `*.dSYM` files instead of the `*.symbols` files generated by the Flutter build. The `*.dSYM` files are located in the `build` folder of your project and not the given `split-debug-info` folder.
 
-## Uploading for Android NDK
+### Uploading for Android NDK
 
-If you are using `sentry_flutter` version earlier than 5.1, native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` for [Android App Bundle](https://developer.android.com/guide/app-bundle) in your `gradle.properties` file.
+The NDK symbols are uploaded by the Sentry Dart Plugin as well if `upload_debug_symbols` option is enabled. As an alternative, you can also use our CLI to [upload source maps for Flutter Web](https://github.com/product/cli/releases/#managing-release-artifacts).
 
-If you use the [Android NDK](/platforms/android/using-ndk/), you must [upload files manually using our CLI](/product/cli/dif/#uploading-files).
+If you are using `sentry_flutter` version earlier than 5.1, native symbolication on Android requires a specific configuration. Refer to [Troubleshooting](/platforms/flutter/troubleshooting/) for more information.
```

Comment:
Link with the `troubleshooting` fragment, so people can go directly to that section.